src/main/java/io/github/darealturtywurty/turtychemistry/mixin/ItemMixin.java

```diff
@@ -0,0 +1,20 @@
+package io.github.darealturtywurty.turtychemistry.mixin;
+
+import io.github.darealturtywurty.turtychemistry.common.block.entity.AnvilBlockEntity;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.ItemLike;
+import net.minecraft.world.level.LevelReader;
+import net.minecraftforge.common.extensions.IForgeItem;
+import org.spongepowered.asm.mixin.Mixin;
+
+@Mixin(Item.class)
+public abstract class ItemMixin implements ItemLike, IForgeItem {
+
+    @Override
+    public boolean doesSneakBypassUse(ItemStack stack, LevelReader level, BlockPos pos, Player player) {
+        return level.getBlockEntity(pos) instanceof AnvilBlockEntity;
+    }
+}
```

Comment:
Is this really necessary? If it is, you should call the super if its not true otherwise RIP compatibility.